app/forms/questionnaire_form.py

```diff
@@ -254,7 +254,8 @@ def get_data(self, answer_id):
         return attr.raw_data[0] if attr.raw_data else ''
 
 
-def get_answer_fields(question, data, error_messages, schema, answer_store, metadata, group_instance, group_instance_id):
+def get_answer_fields(question, data, error_messages, schema, answer_store, metadata,  # pylint: disable=too-many-locals
```

Comment:
this `pylint` should probably go on the next line as it breaks the parameter list

Reply:
Doesn't work on the next line, moved it to top.